common/views/components/IIIFResponsiveImage/IIIFResponsiveImage.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import { classNames } from '../../../utils/classnames';
+import { imageSizes } from '../../../utils/image-sizes';
+import { type IIIFImageService } from '../../../model/iiif';
+import { iiifImageTemplate } from '@weco/common/utils/convert-image-uri';
+
+type Props = {|
+  width: number,
+  height: ?number,
```

Comment:
Is this ever not available? I thought IIIF always gave it to us?

Reply:
Ah, you're probably right; I think this caution pre-dated the change to a IIIF-specific responsive image component.